### PR TITLE
Use Yup for validation

### PR DESCRIPTION
**Pull request by andemerie**

### CODE REVIEW

File: components/auth/LoginForm.js [hunk 1]
```diff
@@ -2,9 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import cn from 'classnames';
 import { Form, Field } from 'formik';
+import * as Yup from 'yup';
```
Comment (Drapegnik) [inline components/auth/LoginForm.js:5]: please, use `import { string, object } from 'yup';`

**Comment by Drapegnik** [comment]
### develop
develop-client

### vd-login-form
vd-login-form-client

File: components/auth/LoginForm.js [hunk 2]
```diff
@@ -48,35 +47,41 @@ const loginFields = {
...
+    .required('forms.required')
+    .when(
+      'isSignUp',
+      (isSignUp, schema) => (isSignUp ? schema.min(7, 'auth.badPassword') : schema)
+    ),
+  passwordAgain: Yup.string().when(
+    'isSignUp',
+    (isSignUp, schema) =>
+      isSignUp
+        ? schema.oneOf([Yup.required('forms.required').ref('password')], 'auth.passwordsNotEqual')
```
Comment (Drapegnik) [inline components/auth/LoginForm.js:81]: is it possible somehow omit this ternary?

**Review by Drapegnik — CHANGES_REQUESTED** [review]

**Comment by Drapegnik** [comment]
lets try [spected](https://github.com/25th-floor/spected):
```
  package        size      minified  gzipped
  spected@0.7.0  72.67 KB  12.42 KB  3.3 KB
```

[Simple React form validation with Formik, Yup and/or Spected](https://itnext.io/simple-react-form-validation-with-formik-yup-and-or-spected-206ebe9e7dcc)

**Comment by UladBohdan** [comment]
@andemerie what about this PR ?

**Comment by Drapegnik** [comment]
[Refactor forms validation](https://trello.com/c/fvZqHXou/280-refactor-forms-validation)

**Comment by UladBohdan** [comment]
I suggest to wait until Oct 27, 2019
Then to celebrate that
And then to close this PR

**Pull request closed without merging**